apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.html.ts

```diff
@@ -0,0 +1,74 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job, User } from '../../../lib/db-types.js';
+
+export function InstructorAIGenerateJobs({
+  resLocals,
+  jobs,
+}: {
+  resLocals: Record<string, any>;
+  jobs: { job: Job; user: User | null }[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          navPage: 'course_admin',
+          navSubPage: 'questions',
+          ...resLocals,
+        })}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Generation job history</div>
+            <div class="table-responsive">
+              <table class="table table-sm table-hover">
+                <thead>
+                  <tr>
+                    <th>Number</th>
+                    <th>Date</th>
+                    <th>User</th>
+                    <th>Status</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${jobs.map(
+                    (row) => html`
+                      <tr>
+                        <td>${row.job.id}</td>
```

Comment:
This is not the job number, and at any rate, I don't think it's very useful to show. Let's drop this column.

Reply:
dropped in a133e9b738f1eb22f09f6fa83561847be451b8eb